src/steamship/app/lambda_handler.py

```diff
@@ -145,6 +153,7 @@ def handler(event: Dict, context: Dict = None) -> dict:
                 host=logging_host,
                 port=logging_port,
                 nanosecond_precision=True,
+                msgpack_kwargs={"default": encode_exception},
```

Comment:
LGTM but this goes over my head so my review is of "sanity check only" quality.

Reply:
It's mainly stack overflow copypasta:

The gist is that the fluent sender uses the msgpack library for efficient data sending (something we may consider down the line!), but msgpack does not by default know how to serialize arbitrary objects, including exceptions.  This line tells fluent to tell msgpack to use the new encode_exception function to attempt to serialize them.